packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -517,6 +517,34 @@ const Breakouts = WebexPlugin.extend({
     });
   },
 
+  /**
+   * Host or cohost update breakout sessions
+   * @param {Object} params
+   * @param {String} params.id
+   * @returns {Promise}
+   */
+  update(params: {id: string}) {
+    if (!params.id) {
+      return Promise.reject(new Error('Missing breakout group id'));
+    }
+    const payload = {...params};
+
+    const body = {
+      ...(this.editLock?.token ? {editlock: {token: this.editLock.token, refresh: true}} : {}),
+      ...{groups: [payload]},
+    };
+
+    return this.request({
+      method: HTTP_VERBS.PUT,
+      uri: this.url,
+      body,
+    }).catch((error) => {
+      return Promise.reject(
+        boServiceErrorHandler(error, 'Breakouts#update --> Edit lock token mismatch')
```

Comment:
issue: I can't see a test checking this reject

Reply:
@othellashao are you sure the message here 'Breakouts#update --> Edit lock token mismatch' is correct? Given that other errors are possible perhaps just "Breakouts#update" might be better